README.md

```diff
@@ -41,13 +41,14 @@ If you'd like to also classify nodes in ServiceNow's CMDB, then the module:
 **Note:** If you're interested in both the ServiceNow CMDB external data and node classification features of this module, then you can just read the [classification](#ServiceNow-node-classification) section since that depends on the CMDB external data feature.
 
 #### ServiceNow CMDB as trusted external data
-Apply the `servicenow_cmdb_integration::trusted_external_command` class on all of your compilers. This looks something like:
+Apply the `servicenow_cmdb_integration::trusted_external_command` class on all of your compilers. You may include a user and password or an oauth token This looks something like:
```

Comment:
I'd mention that if an `oauh_token`'s provided, then that will take precedence over username/password.

I also forgot to mention hiera-eyaml encryption here. Added "Double-check README" as a TODO to the doc.

Reply:
I don't think we need to mention this anymore, but I also think README should be updated after "Getting Started" is finalized so don't worry about changing anything here.